### PR TITLE
Dev improvements

### PR DESCRIPTION
**Pull request by downiec**
This PR includes some updates to the Selenium tests suite and also includes updates to the package.json by moving some of the scripts out and incorporating their functionality within a 'tasksfile.ts'. The tasksfile provides better documentation, readability, and improved functionality for running automated development tasks when compared to running scripts within the package.json.

To run a task, you would use: 'npx task task_name'
By running: 'npx task --help', you will see documentation on available automated tasks and what they do. And to get help for a specific task, you would use: 'npx task task_name --help'

Included in the task file are functions for setting up and running selenium automated testing.

Current available tasks:
**checkVersion**                    - Compares package.json version with the version published on npm and exits with error if the published version is newer.
**format**                          - Format source files using prettier-tslint
**installTestTools**                - Installs dependencies for running Selenium tests locally.
**lint**                            - Performs linting operations on source files.
**test**                    - Runs Selenium automated tests locally. If no tests are specified, all available tests will be run. If no browser options are specified, both browsers will be used. 
**updateTags**                      - Will find and update all className tags in source files.

### CODE REVIEW

**Review by muryanto1 — COMMENTED** [review]
@downiec changes on the tests look ok.  I assume you intentionally comment out the running of the tests with firefox?

**Review by muryanto1 — APPROVED** [review]

**Comment by downiec** [comment]
Yes the firefox tests are commented out for now, until I can figure out why they won't pass on circleci (they do pass when run locally).

File: src/components/InputModal.tsx [hunk 1]
```diff
@@ -236,6 +234,7 @@ export default class InputModal extends React.Component<
     if (code === 13) {
       if (this.state.isValid) {
         this.props.onModalClose(this.state.input, this.state.savedInput);
+        await this.setState({ modalOpen: false, input: "" });
```
Comment (sterlingbaldwin) [inline src/components/InputModal.tsx:237]: why block here?

**Review by sterlingbaldwin — APPROVED** [review]
Looks good to me, nothing big in here from a user standpoint.

File: src/components/InputModal.tsx (hunk 1 above)
Comment (downiec) [reply src/components/InputModal.tsx:237]: Good point, the await there is probably not needed. I will clean this and other await statements in next PR.

**Pull request merged**